spec/api/api.yaml

```diff
@@ -833,11 +836,12 @@ paths:
         - API
       summary: Starts a Multipeer replicator
       description: |-
-        Starts a P2P listener so that the device can act as the passive side of a P2P replication.
+        Starts a multipeer replicator to create a mesh of connected clients.
       operationId: startMultipeerReplicator
       requestBody:
         description: |-
-          The request object containing the collections to share and, optionally, the port to listen on.
+          The request object containing the collections to share and the peer group ID.  Optionally,
+          it can contain an authenticator as well.  The default (null) behavior is accept all connections.
```

Comment:
Is identity optional or mandatory?

Reply:
Optional. It’s not in the list of required fields.

Reply:
Made it mandatory but yes a new self signed one will be generated every time, just on the TDK client side now